### ISSUE TITLE
add a switch button or make it clickable  on #exchange to switch between base asset and counter asset.

### ISSUE THREAD

**Issue by manran** [irisli/stellarterm#7]
it will be more convenient if we can switch between base asset and counter asset with one click.

**Comment by sacarlson** [comment]
Yes this is the feature I need that should be easy to do,  I presently manually change the URL and reload to switch my view now.  If you don't fix this soon I will add this to my branch.
